common/catalog/docker/docker.bom

```diff
@@ -192,7 +192,7 @@ brooklyn.catalog:
           label: "Docker Port"
           description: |
             The TCP port for Docker to listen on
-          type: port
+          type: integer
```

Comment:
Although this is changed, none of the `attributeWhenReady("docker.port")` uses have been updated